src/main/java/com/github/cameltooling/lsp/internal/completion/modeline/TraitProperty.java

```diff
@@ -28,6 +28,13 @@
 	public CompletionItem createCompletionItem() {
 		CompletionItem completionItem = new CompletionItem(name);
 		completionItem.setDocumentation(description);
+		if (defaultValue != null) {
+			if("int".equals(type) && defaultValue instanceof Double) {
+				completionItem.setInsertText(name+"="+((Double)defaultValue).intValue());
```

Comment:
how about using String.format instead of multiple concatenations?

Reply:
what is the advantage?

https://redfin.engineering/java-string-concatenation-which-way-is-best-8f590a7d22a8

`Avoid using String.format() when possible. It is slow and difficult to read when you have more than two variables.`